src/app/App.test.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { App } from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+// test('renders learn react link', () => {
```

Comment:
Taka sugestia na przyszłość - lepiej nie commitować zakomentowanego kodu, to wygląda naprawdę źle :) Zamiast tego lepiej usunąć cały plik. Zawsze możesz do niego wrócić używając gita